ci/docker/Dockerfile-benchmark

```diff
@@ -0,0 +1,22 @@
+FROM frolvlad/alpine-glibc
+
+CMD ["./benchmarks" "--help"]
+ENV RUNFILES_DIR=/usr/local/bin/benchmarks/benchmarks.runfiles
+ENV ENVOY_PATH=envoy
+ENV ENVOY_IP_TEST_VERSIONS=v4only
+WORKDIR /usr/local/bin/benchmarks
+
+COPY benchmarks /usr/local/bin/benchmarks/
+
+RUN apk add docker openrc
```

Comment:
should this point to a specific versions to pin them?

Reply:
Yep.  Let me spend some more time on this and I will update.